tests/components/sensor/test_dsmr.py

```diff
@@ -0,0 +1,56 @@
+"""Test for DSMR components."""
+
+import asyncio
+from decimal import Decimal
+
+from homeassistant.bootstrap import async_setup_component
+from tests.common import assert_setup_component
+
+
+@asyncio.coroutine
+def test_default_setup(hass, monkeypatch):
+    """Test the default setup."""
+    from dsmr_parser.obis_references import (
+        CURRENT_ELECTRICITY_USAGE,
+        ELECTRICITY_ACTIVE_TARIFF,
+    )
+    from dsmr_parser.objects import CosemObject
+
+    config = {'platform': 'dsmr'}
+
+    telegram = {
+        CURRENT_ELECTRICITY_USAGE: CosemObject([
+            {'value': Decimal('0.1'), 'unit': 'kWh'}
+        ]),
+        ELECTRICITY_ACTIVE_TARIFF: CosemObject([
+            {'value': '0001', 'unit': ''}
+        ]),
+    }
+
+    # mock queue for injecting DSMR telegram
+    queue = asyncio.Queue(loop=hass.loop)
+    monkeypatch.setattr('asyncio.Queue', lambda: queue)
```

Comment:
btw, have you considered using https://github.com/klen/pylama for the project? It combines a lot of the linting tools used in this project into a single consistent interface.